presto-spi/src/main/java/io/prestosql/spi/predicate/SortedRangeSet.java

```diff
@@ -88,9 +89,7 @@ static SortedRangeSet of(Range first, Range... rest)
     {
         List<Range> rangeList = new ArrayList<>(rest.length + 1);
         rangeList.add(first);
-        for (Range range : rest) {
-            rangeList.add(range);
-        }
+        rangeList.addAll(Arrays.asList(rest));
```

Comment:
static import `asList`

FYI
there is `java.util.Collections#addAll(Collection, T... elements)` for that

(no change requested)